lnrpc/lightning.proto

```diff
@@ -2848,6 +2848,11 @@ message QueryRoutesRequest {
     */
     repeated lnrpc.RouteHint route_hints = 16;
 
+    // An optional blinded path to reach the destination via. Note that the
+    // introduction node included in this path *must* be the same as the target
+    // pubkey provided, and there must be at least two hops in the path.
```

Comment:
NIT: I think now with the introduction node allowed to be the destination of a blinded payment a single hop is allowed (see also description of `blinded_hops` under `BlindedRoute` below).